src/components/Footer.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Todo } from '../types/Todo';
+import cn from 'classnames';
+
+type Props = {
+  todos: Todo[];
+  onFilter: React.Dispatch<React.SetStateAction<string>>;
+  filter: string;
+  handleDeleteAllCompleted: (currentsTodo: number[]) => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  onFilter,
+  filter,
+  handleDeleteAllCompleted,
+}) => {
+  const hasCompletedTodos = todos
+    .filter(todo => todo.completed)
+    .map(todo => todo.id);
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {todos.filter(todo => !todo.completed).length} items left
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={cn('filter__link', { selected: filter === 'All' })}
+          data-cy="FilterLinkAll"
+          onClick={() => onFilter('All')}
+        >
+          All
+        </a>
+
+        <a
+          href="#/active"
+          className={cn('filter__link', { selected: filter === 'active' })}
+          data-cy="FilterLinkActive"
+          onClick={() => onFilter('active')}
+        >
+          Active
+        </a>
+
+        <a
+          href="#/completed"
+          className={cn('filter__link', { selected: filter === 'completed' })}
+          data-cy="FilterLinkCompleted"
+          onClick={() => onFilter('completed')}
+        >
```

Comment:
Use `Object.values(your created enum)` and render these options with `map()` method